ddmrp/models/stock_picking.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class StockPickgin(models.Model):
+    _inherit = "stock.picking"
+
+    def action_stock_buffer_open(self):
+        """Open a sock.buffer list related to products of the stock.picking."""
+        product_ids = self.mapped("move_lines.product_id.id")
+        return {
+            "type": "ir.actions.act_window",
+            "name": "Stock Buffers",
+            "res_model": "stock.buffer",
+            "view_mode": "tree,form",
+            "domain": [("product_id", "in", product_ids)],
```

Comment:
it is not that easy, there can be more than one buffer per product, this should be implemented similar as how it is done for purchase orders, where each line can be linked to a buffer and its execution estatus is shown for each line in the PO.

Reply:
@LoisRForgeFlow Thanks for your comment. Oh, I am not surprised that it could be more complex.
I guess you mean the `_find_buffer_link` function and the `buffer_ids` Many2many on `purchase.order.line`.
Will try to get some inspiration from those.

Reply:
Yes, exactly. Thanks for contributing :)

Reply:
I only improved the domain a little and will try to get some more information tomorrow

Reply:
So the idea of this smart button is not to have access to the exact buffers related to the transfer. But to give a view of all the buffers related to the product in the transfer. As to troubleshoot a situation.
But will also try to implement the `On hand/TOR` on the `stock.move` in the `Operation` tab.